src/clarity-angular/popover/signpost/_signposts.clarity.scss

```diff
@@ -12,19 +12,19 @@
             cursor: pointer;
         }
 
-        .signpost-action {
-            min-width: $clr_baselineRem_1;
-            margin: 0;
-            padding: 0;
+        .signpost-trigger {
             color: $clr-dark-midtone-gray;
-            outline: none;
 
-            clr-icon {
-                width: $clr_baselineRem_1;
-                height: $clr_baselineRem_1;
+            &.btn-icon {
+                border: none;
+            }
+
+            &.active, &:hover {
+                color: $clr-action-blue;
+                background-color: transparent;
```

Comment:
same for this. no background if the trigger is a primary button? I am assuming that UX mentioned that signposts are used over actionable icons only? which means a transparent button with just the icon.

Reply:
I forget what caused me to add that. I'll remove it. 
Our docs don't require SignpostTriggers to be actionable icons only. Triggers could be anything (span, label etc ...)